t/command/yq/functions.t

```diff
@@ -21,8 +22,13 @@ subtest 'grep( EXPR )' => sub {
 };
 
 subtest 'empty' => sub {
-    my $out = $class->filter( 'empty', $doc );
+    my $out;
+    my ( $stdout, $stderr, $exit ) = capture {
+        $out = $class->filter( 'empty', $doc );
+    };
     isa_ok $out, 'empty';
+    like $stderr, qr/empty does not take arguments/,
```

Comment:
What's making this test warn is the code in ETL::Yertl::Transform::Yq is wrong: The filter `'empty'` is not passing an argument to the `empty` function, so it should not be throwing this warning. The test is good, but the condition needs to be switched from `like` to `unlike`, and the parsing bug fixed. I can accept this change as-is and do the fix myself if you'd like.

Reply:
Yeah, after I pushed I was thinking that there should be one test that throws and captures a warning and one that is formulated so as to not throw any warning.  I'll leave that to you.